images/external.md

```diff
@@ -18,6 +18,14 @@ such as [`jupyter/r-notebook`](https://jupyter-docker-stacks.readthedocs.io/en/l
 and [`jupyter/datascience-notebook`](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-datascience-notebook),
 so you can immediately run R on Jupyter.
 
+### [b-data/jupyterlab-r-docker-stack](https://github.com/b-data/jupyterlab-r-docker-stack)
+
+Multi-arch (`linux/amd64`, `linux/arm64/v8`) docker images based on
+[Debian](https://hub.docker.com/_/debian) including
+[code-server](https://github.com/coder/code-server) (VS Code in the browser).
```

Comment:
Perhaps upper case "VS Code" is recognized as "VisualStudio Code"?

https://github.com/rocker-org/website/blob/ed620eb0d21d01b71d97054ec94d7b6e835f0d43/use/editor.md?plain=1#L72

```suggestion
[code-server](https://github.com/coder/code-server) (vscode in the browser).
```

Reply:
[code-server](https://github.com/coder/code-server) is based on [vscode](https://github.com/microsoft/vscode) (Code - OSS).